### ISSUE TITLE
Accessibility: Focusable elements does not meet size requirements

### ISSUE THREAD

**Issue by MarkoTP** [microsoft-search/pnp-modern-search#4027]
**Version used**
4.13.1

**Describe the bug**
Accessibility:
A few PnP Modern Search issues were identified during the accessibility audit.
Some focusable elements on the PnP Modern Search Results webpart does not meet size requirements 24px x 24px.
WCAG 2.2, 2.5.8

**To Reproduce**
Detailed steps to reproduce the behavior:
Configure the search page as you like with search box and results web parts. 
Execute a search that returns filterable results.
Execute a search that returns enough results to paginate multiple pages.

**Expected behavior**
The focusable area of pagination numbers, next and last buttons should be at least 24px x 24px.

**Desktop (please complete the following information):**
Browser: Chrome, Version 129.0.6668.90 (Official Build) (64-bit)
